lms/templates/ccx/schedule.underscore

```diff
@@ -1,49 +1,86 @@
+  <div align="right">
+    <button id="ccx_expand_all_btn" class="ccx-button-link"><i class="fa fa-expand"></i> Expand All</button>
+    <button id="ccx_collapse_all_btn" class="ccx-button-link"><i class="fa fa-compress"></i>Collapse All</button>
+  </div>
+  <br/>
   <table class="ccx-schedule">
     <thead>
       <tr>
         <th><%- gettext('Unit') %></th>
         <th><%- gettext('Start Date') %></th>
         <th><%- gettext('Due Date') %></th>
-        <th><a href="#" id="remove-all">
+        <td><button id="remove-all" class="ccx-button-link">
           <i class="fa fa-remove"></i> <%- gettext('remove all') %>
-        </a></th>
+        </button></td>
       </tr>
     </thead>
     <tbody>
       <% _.each(chapters, function(chapter) { %>
         <tr class="chapter collapsed" data-location="<%= chapter.location %>" data-depth="1">
-          <td class="unit">
-            <a href="#"><i class="fa fa-caret-right toggle-collapse"></i></a>
-            <%= chapter.display_name %>
+          <td class="unit" aria-label="Link chapter <%= chapter.display_name %>">
+            <button class="toggle-collapse ccx-button-link" aria-expanded="false"><i class="fa fa-caret-right"></i></button>
+            <span aria-label="Section <%= chapter.display_name %>"><%= chapter.display_name %></span>
```

Comment:
i wouldn't set an aria-label on a span.  I think you should add offscreen (sr) text in the button text.